src/mgmt/srv_drpc.c

```diff
@@ -406,8 +406,11 @@ static int pool_create_fill_resp(Mgmt__PoolCreateResp *resp, uuid_t uuid, d_rank
 
 	D_DEBUG(DB_MGMT, "%d service replicas\n", svc_ranks->rl_nr);
 
-	rc = ds_mgmt_pool_query(uuid, svc_ranks, &enabled_ranks, NULL, NULL, &pool_info, NULL, NULL,
+	rc = ds_mgmt_pool_query(uuid, svc_ranks, &enabled_ranks, NULL, NULL,
+				daos_getmtime_coarse() + 60 * 1000, &pool_info, NULL, NULL,
```

Comment:
Are there any scenarios where the timeout will not be sufficient and cause an intermittent type of failure?

Reply:
One reason behind shortening this timeout is to leave more time for the cleanup.

Reply:
I'm happy with either value just wanted to ask the question. thanks for the explanation